assets/src/sass/blocks/_banner.scss

```diff
@@ -33,6 +33,8 @@
 
 	&__image {
 		@include absolute-full-cover();
+		content: unset !important; // Overwritting content property from the mixin - see #799
```

Comment:
Since this is declared after the mixin, it should override the content property from the mixin without this !important.

Its not an issue one way or the other though...

Reply:
@goldenapples I thought the same, but I decided to add it just in case. Do you think that it´s safe to remove it?